alf/environments/suite_dmc.py

```diff
@@ -0,0 +1,174 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import alf
+from alf.environments.alf_environment import AlfEnvironment
+from alf.environments.suite_gym import wrap_env
+from unittest.mock import Mock
+
+try:
+    import dmc2gym
+except ImportError:
+    # create 'dmc2gym' as a mock to not break python argument type hints
+    dmc2gym = Mock()
+
+
+def is_available():
+    return not isinstance(dmc2gym, Mock)
+
+
+class AlfEnvironmentDMC2GYMWrapper(AlfEnvironment):
+    """AlfEnvironment wrapper forwards calls to the given environment."""
+
+    def __init__(self, env):
+        """Create an ALF environment base wrapper.
+
+        Args:
+            env (AlfEnvironment): An AlfEnvironment instance to wrap.
+
+        Returns:
+            A wrapped AlfEnvironment
+        """
+        super(AlfEnvironmentDMC2GYMWrapper, self).__init__()
+        self._env = env
+
+    def __getattr__(self, name):
+        """Forward all other calls to the base environment."""
+        if name.startswith('_'):
+            raise AttributeError(
+                "attempted to get missing private attribute '{}'".format(name))
+        return getattr(self._env, name)
+
+    @property
+    def batched(self):
+        return self._env.batched
+
+    @property
+    def batch_size(self):
+        return self._env.batch_size
+
+    @property
+    def num_tasks(self):
+        return self._env.num_tasks
+
+    @property
+    def task_names(self):
+        return self._env.task_names
+
+    def _reset(self):
+        reset_result = self._env.reset()
+        reset_result = reset_result._replace(env_info={})
+        return reset_result
+
+    def _step(self, action):
+        step_result = self._env.step(action)
+        step_result = step_result._replace(env_info={})
+        return step_result
+
+    def get_info(self):
+        return self._env.get_info()
+
+    def env_info_spec(self):
+        return self._env.env_info_spec()
+
+    def time_step_spec(self):
+        return self._env.time_step_spec()
+
+    def observation_spec(self):
+        return self._env.observation_spec()
+
+    def action_spec(self):
+        return self._env.action_spec()
+
+    def reward_spec(self):
+        return self._env.reward_spec()
+
+    def close(self):
+        return self._env.close()
+
+    def render(self, mode='rgb_array'):
+        return self._env.render(mode)
+
+    def seed(self, seed):
+        return self._env.seed(seed)
+
+    def wrapped_env(self):
+        return self._env
+
+
+@alf.configurable
+def dmc_loader(environment_name,
+               domain_name='cheetah',
+               task_name='run',
+               seed=1,
+               from_pixels=True,
+               pre_transform_image_size=100,
```

Comment:
Why the prefix "pre_transform_"? It seems to me its the actual image size from the environment?